packages/bfx-containers/src/ufx-bfx-containers.scss

```diff
@@ -1,4 +1,4 @@
-@import "~@ufx-ui/core/dist/css/ufx-core.bundle.scss";
+@import "../../core/dist/css/ufx-core.bundle.scss";
```

Comment:
why are we changing this ? I think this may create path issues, if someone uses standalone `@ufx-ui/bfx-containers` package and import row sass file src/ufx-bfx-containers.scss

Reply:
because otherwise the styles from the VirtualTable are not impoted neither on storybook nor on the npm build

Reply:
![Screenshot from 2021-05-13 14-16-05](https://user-images.githubusercontent.com/35810911/118118876-0ea87f00-b3dd-11eb-9df2-8ab7a0f135ab.png)